sources/constraint-definition.asciidoc

```diff
@@ -436,7 +436,7 @@ If left undefined, the default value for [methodname]`@OverridesAttribute.name`
 
 [NOTE]
 ====
-[tck-testable]#A composing constraint can itself be a composed constraint. In this case, attribute values are overridden recursively according to the described rules.# Note however, that a forwarding rule (as defined by [classname]`@OverridesAttribute`) is only applied to the direct composing constraints.
```

Comment:
Let's discuss an example to make sure we are on the same page on the terms:

    @NotNull
    @Size(min=1)
    public @interface NotEmpty {
        ...
    }

Here I'd consider `@NotEmpty` as the composed constraint and `NotNull`/`@Size` as the composing constraints. Of course `@NotNull` could be used to compose other constraints than `@NotEmpty` (which seems to be what you are saying), but that's not what this section is about IIUC. Instead it's there to say that `@NotNull` and `@Size` could be composed constraints themselves. In that light, the original version was correct. Does that make sense?